cmd/internal/cli/key_import.go

```diff
@@ -27,9 +29,18 @@ var KeyImportCmd = &cobra.Command{
 	Example: docs.KeyImportExample,
 }
 
+var keyImportWithNewPassword bool
+var keyImportWithNewPasswordFlag = cmdline.Flag{
+	ID:           "keyImportWithNewPasswordFlag",
+	Value:        &keyImportWithNewPassword,
+	DefaultValue: false,
+	Name:         "with-newpassword",
```

Comment:
Maybe just `new-password` will be better?

```suggestion
	Name:         "new-password",
```